package.json

```diff
@@ -18,6 +21,7 @@
     "d3": "^7.3.0",
     "gulp": "4.0.2",
     "gulp-append-prepend": "1.0.8",
+    "jest": "26.6.0",
```

Comment:
Was this necessary? I believe CRA has Jest built in.

Reply:
Done